makePoplogCommander.sh

```diff
@@ -422,8 +459,15 @@ cat << \****
         } else if ( strcmp( "--help", argv[1] ) == 0 ) {
             printUsage( argc - 2, &argv[2] );
             return EXIT_SUCCESS;
+        } else if ( strcmp( "--run", argv[1] ) == 0 ) {
+            //  We want to force overwrites.
```

Comment:
Comment is wrong? Or the one below is wrong

Reply:
That's a flag-naming issue, which I will fix. I was trying to keep to the meaning of the 3rd parameter to setenv, which is called `overwrite` (but might be better named `force`). The flags are named in terms of adding features that increase flexibility at the expense of security.

 - INHERIT_ENV, is a feature to prefer the external envvar.
 - RUN_INIT_P, is a feature to run `$poplib/init.p` (or other files described in INITIAL)
 
 The `--run` flag is a more secure mode and therefore passes the value 0, which I will give a logical name such a `PREFER_SAFETY`. And also I can define `INHERIT_ENV|RUN_INIT_P` as `PREFER_FLEXIBILITY`.
 
 On balance I think my answer is, yes, the comment is misleading. I will fix.

Reply:
Done